.github/workflows/python-package.yml

```
@@ -0,0 +1,201 @@
+name: Python package
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ master ]
```

The main branch is `main` here:
```suggestion
    branches: [ main ]
```